Makefile.in

```diff
@@ -228,6 +228,13 @@ config_files: ocaml-stage1-config.status
 	/bin/echo -n "( `grep \"^SHAREDLIB_CFLAGS=\" ocaml/Makefile.config \
 		| sed 's/^SHAREDLIB_CFLAGS=//'` )" > sharedlib_cflags.sexp
 
+## For targets not supported in backend, we can install the result of stage0
+.PHONY: install_stage0
+install_stage0: stage0
+	rsync --copy-links --chmod=u+rw,go+r -r $(stage0_prefix)/* $(prefix)
```

Comment:
I think before this `rsync` line we need a `mkdir -p` of the prefix directory (see the usual installation rules elsewhere in this file).

Reply:
Thank you for spotting this! Fixed.